docs/getting-started/install-endpoint.asciidoc

```diff
@@ -131,15 +141,51 @@ Behavioral ransomware prevention detects and stops ransomware attacks on Windows
 * **Prevent** (Default): Detects ransomware on the host, blocks it from executing, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
 +
 TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
 [role="screenshot"]
-image::images/install-endpoint/ransomware-protection.png[]
+image::images/install-endpoint/ransomware-protection.png[Detail of ransomware protection section.]
+
+[discrete]
+[[memory-protection]]
+=== Memory manipulation protection
+
+Memory manipulation protection detects and stops in-memory threats on Windows systems, such as shellcode injection, which are used to evade traditional file-based detection techniques.
+
+Memory manipulation protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, memory manipulation protection will be disabled by default.
+
+Memory manipulation protection levels are:
+
+* **Detect**: Detects memory manipulation activity on the host and generates an alert. The {agent} will **not** block the in-memory activity. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects memory manipulation activity on the host, blocks it from executing, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
+
+[role="screenshot"]
+image::images/install-endpoint/memory-protection.png[Detail of memory protection section.]
+
+[discrete]
+[[behavior-protection]]
+=== Behavior protection
+
+Behavior protection detects and stops threats by monitoring the behavior of system processes for suspicious activity. Behavioral signals are much more difficult for adversaries to evade than traditional file-based detection techniques.
+
+Behavior protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, behavior protection will be disabled by default.
+
+Behavior protection levels are:
+
+* **Detect**: Detects suspicious behavior on the host and generates an alert. The {agent} will **not** block suspicious behavior. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects suspicious behavior on the host, blocks it from executing, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
```

Comment:
this default will be `Elastic Security {action} {rule}` for Memory and Behavior protections

Reply:
Same as memory manipulation: I've seen this change in a 8.0.0 test server, but the UI still has `Elastic Security {action} {filename}` in the 7.15 BC4 test server I'm looking at. Is `{action} {rule}` a planned change for BC >4 but still in 7.15?

Reply:
@joepeeples yes, this will be a change appearing in the BC6 build.